ownCloud/Client/Actions/Actions+Extensions/DuplicateAction.swift

```diff
@@ -20,11 +20,14 @@ import Foundation
 
 import ownCloudSDK
 
-class DuplicateAction : Action {
+class DuplicateAction : Action, OCQueryDelegate {
 	override class var identifier : OCExtensionIdentifier? { return OCExtensionIdentifier("com.owncloud.action.duplicate") }
 	override class var category : ActionCategory? { return .normal }
 	override class var name : String? { return "Duplicate".localized }
 	override class var locations : [OCExtensionLocationIdentifier]? { return [.moreItem, .moreFolder, .toolbar] }
+	var query : OCQuery?
+	var localizedCopy = "copy".localized
```

Comment:
@hosy don't think it is worth storing as extra property.. also should be constant and not `var`. And also, shall we call it` "blah copy x"`? macOS for instance just uses numbers.